portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -583,6 +583,25 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, cancer_study_
 	'getGeneticProfileIds': function () {
 	    return this.genetic_profile_ids;
 	},
+	'getDefaultGeneticProfileId': function () {
+			var gp_ids = this.getGeneticProfileIds();
+			var chosen_ids = [];
+			for (var i = 0; i < gp_ids.length; i++) {
+				if (~gp_ids[i].indexOf('Zscores') && ~gp_ids[i].indexOf('mrna')) {
```

Comment:
oh interesting, good call. Yes I wholeheartedly agree with @fedde-s, dealing with -1 more explicitly is definitely easier for people in general (definitely me) to read.